### PR TITLE
fill out research plan phase 1

### PR DESCRIPTION
**Pull request by kilbergr**
# [Document and share interview questions for phase 1 of research #12](https://github.com/OHS-Hosting-Infrastructure/infrastructure/issues/12)
and
# [Document phase 1 of research plan](https://github.com/OHS-Hosting-Infrastructure/infrastructure/issues/11)

Changes proposed in this pull request:

- Add a Phase 1 of a research plan
- I'm pretty unsure of the timeline section, so....just leaving it there.

### CODE REVIEW

File: docs/user-research/research-plan-phase1.md [hunk 1]
```diff
@@ -1,79 +1,231 @@
...
 
 The ECLKC hosting infrastructure is hosted in AWS and serves the following applications:
 
 * ECLKC
 * National Centers
-  * iDP
-  * (more)
+  * iDP Learning Management System
+  * Coaching Companion
+  * iLookOut
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:20]: iLookout is not actually an app under the national centers and is an application that's integrated with ipd.

File: docs/user-research/research-plan-phase1.md [hunk 2]
```diff
@@ -1,79 +1,231 @@
...
+* HSICC
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:21]: HSICC is a team and not an application.

File: docs/user-research/research-plan-phase1.md [hunk 3]
```diff
@@ -1,79 +1,231 @@
...
+
+Ultimately, there's hope to include these applications as well:
+
+* National Centers
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:25]: These are already in our infrastructure but may be sub applications of ECLKC.

File: docs/user-research/research-plan-phase1.md [hunk 4]
```diff
@@ -1,79 +1,231 @@
...
+
 
 We believe there are modificiations to technology and practice that can allow for self-service of the application deployments while maintaining ATO.
 
 This is also our pilot to see if we can define practice and standards for technical teams working with OHS outside of the ECLKC infrastructure.
 
 ## Goals
 
-> When writing your [research goals](https://ux-guide.18f.gov/research/plan/#goals), use reality-oriented words like “describe”, “evaluate”, or “identify,” and avoid intangible ones like “understand” or “explore.” Consider how this research will help reduce risk and inform decisions.
+We have the following goals for the initial round of research:
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:41]: Great description of our goals!

File: docs/user-research/research-plan-phase1.md [hunk 5]
```diff
@@ -1,79 +1,231 @@
...
+1. Who else should we speak to?
+1. Would you be interested in someone from our team embedding with yours for a few days?
+1. Would you be interested in coming to demos of work we’ve completed?
 
 ## Method(s)
 
-> **Instructions:** Choose some [methods](https://ux-guide.18f.gov/research/plan/#methods) that are appropriate for meeting the goals and answering research questions.
+We have already conducted the stakeholder mapping excercise (available [here](https://miro.com/app/board/o9J_lCdf5Iw=/?moveToWidget=3074457359393677819&cot=14)).
+
+We will be completing a Process download workshop with at least the HSICC team and likely the Coaching Companion and iLookOut teams. An example of that workshop is available [here](https://miro.com/app/board/o9J_lCdf5Iw=/?moveToWidget=3074457359402237068&cot=14). We'll update this plan with links to the workshops as they are completed.
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:181]: When you write iLookOut, do you mean ipd?

File: docs/user-research/research-plan-phase1.md [hunk 6]
```diff
@@ -1,79 +1,231 @@
...
+| Collaborative analysis   |  3 days |
+| Collaborative synthesis   |  3 days |
+| Summary/outputs   |  3 days |
 
 
 ## Participants and recruiting
 
-> Describe at a high-level [who should participate in this study](https://ux-guide.18f.gov/research/plan/#participants-and-recruiting), and how you’ll recruit them. Consider how the team will communicate with participants, for example [by email](https://ux-guide.18f.gov/resources/#email-templates) and how the team will get [informed consent](https://ux-guide.18f.gov/research/do/#getting-informed-consent) from participants. See the [legal section](https://ux-guide.18f.gov/research/legal/) of the UX Guide for more information.
+We expect to work closely with members of the HSICC team, Coaching Companion, iLookOut team. The last we'll specifically be focusing on issues around the shared Central Authentication Service (CAS). Alana Buroff, Information Systems and Communications Team lead, completed a stakeholder exercise and identified who we should speak to for the first round. She will be sending the introduction emails to stakeholder/users targeted in this user research plan. We will then follow up with the teams.
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:213]:
```suggestion
We expect to work closely with members of the HSICC team, Coaching Companion, iPD team. With the last team we'll specifically be focusing on issues around the shared Central Authentication Service (CAS). Alana Buroff, Information Systems and Communications Team lead, completed a stakeholder exercise and identified who we should speak to for the first round. She will be sending the introduction emails to stakeholder/users targeted in this user research plan. We will then follow up with the teams.
```

I think this is what you meant?

File: docs/user-research/research-plan-phase1.md [hunk 7]
```diff
@@ -1,79 +1,231 @@
...
 
 ## Ethics considerations
 
-> Document the [ethical principles or concerns](https://ux-guide.18f.gov/research/plan/#ethical-considerations) that influence this research approach. Discuss as a team the [biases](https://ux-guide.18f.gov/research/bias/) that could influence the work.
-
+We will be recording all these conversations via Zoom and sharing them with our partners at OHS and 18F.
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:217]: ** with consent?

**Review by eeeady — CHANGES_REQUESTED** [review]
Looks good so far! I do think you've mixed up iLookout for iPD though?

File: docs/user-research/research-plan-phase1.md [hunk 8]
```diff
@@ -1,79 +1,226 @@
...
 
 ## Expected outcomes
 
-> Document what [outputs and outcomes](https://ux-guide.18f.gov/research/plan/#outputs-and-outcomes) the team expects from this research.
+At the end of this research round, we should have:
+ * a stakeholder map,
+ * multiple process download diagrams,
+ * basic architecture diagrams for at least the HSICC team,
+ * user research synthesis, and
+ * part of a product roadmap.
```
Comment (ssatagopan) [inline docs/user-research/research-plan-phase1.md:221]: Should we adjust this to be "list of possible changes to prioritize in product roadmap" and then we go through prioritization exercise (bang for buck exercise) to construct actual roadmap?

**Review by rahearn — COMMENTED** [review]
I like this list of questions. I'd like @matt-dobson to also give this a review as a Researcher.

File: docs/user-research/research-plan-phase1.md [hunk 9]
```diff
@@ -1,79 +1,226 @@
...
 
 * ECLKC
+* HSICC
 * National Centers
-  * iDP
-  * (more)
+  * iDP Learning Management System (includes iLookOut)
+  * Coaching Companion
+  * Early Educator Central
+  * Mobile Applications
```
Comment (jduss4) [inline docs/user-research/research-plan-phase1.md:22]: Would you mind indenting the mobile apps (like the ECLKC Resource App, if you also manage that?, ELOF*, Ready-DLL, etc) ?  At a glance I didn't know that those were all mobile applications until I looked them up to read about them.

File: docs/user-research/research-plan-phase1.md [hunk 10]
```diff
@@ -1,79 +1,226 @@
...
+1. Is there anything you wish we asked about?
+1. Is there anything we’ve missed?
+1. Who else should we speak to?
+1. Would you be interested in someone from our team embedding with yours for a few days?
+1. Would you be interested in coming to demos of work we’ve completed?
 
 ## Method(s)
 
-> **Instructions:** Choose some [methods](https://ux-guide.18f.gov/research/plan/#methods) that are appropriate for meeting the goals and answering research questions.
+We have already conducted the stakeholder mapping excercise (available [here](https://miro.com/app/board/o9J_lCdf5Iw=/?moveToWidget=3074457359393677819&cot=14)).
```
[the remark this answers is not in the document]
Comment (kilbergr) [reply docs/user-research/research-plan-phase1.md:174]: Thanks! Always looking for marginal gains!

File: docs/user-research/research-plan-phase1.md [hunk 11]
```diff
@@ -1,79 +1,226 @@
 # Research plan - Task 1
 
-DRAFT - last updated by Eady on 04/13/2021
-
 This research plan uses the [18F research plan template](https://github.com/18F/ux-guide/blob/master/_pages/resources/research-plan.md).
 
 ## Background
 
-> Describe factors that the research will need to account for, including any shared beliefs or forces motivating the research itself. Summarize any past research or information that shape the research context.
+These are the goals for this project as defined by OHS:
```
[the remark this answers is not in the document]
Comment (kilbergr) [reply docs/user-research/research-plan-phase1.md:7]: Those I lifted from Alana during our Intro to OHS call. She had a document with them. These are my notes from that call: https://docs.google.com/document/d/13U7Swfn8BC0k9Z3KtFRlKVx43rbS6XnAfwx-rxSG7CU/edit

File: docs/user-research/research-plan-phase1.md [hunk 12]
```diff
@@ -1,79 +1,226 @@
...
+* Determine current application needs regarding development and deployment
+* Describe what success looks like for the interviewee
 
 ## Research questions
 
-> What do you want to learn to make better evidence-based decisions? [Good research questions](https://ux-guide.18f.gov/research/plan/#research-questions) are specific, actionable, and practical; they usually center around users' experiences and needs.
-
+### Intro
+
+Thank you for joining us today. My name is XXX and I’m an XXX [add your name and position] for Truss Works. We’re contracting with OHS to help manage and configure infrastructure for the apps under the OHS umbrella, support automation where possible, and to engender an internal devsecops community of practice. We wanted to have a conversation with you today to better understand your role, the challenges you face, and your experiences with current tools, as well as your day-to-day work. The information you provide will inform what we may build.
```
[the remark this answers is not in the document]
Comment (kilbergr) [reply docs/user-research/research-plan-phase1.md:48]: That's a good question. I'll add a mini def.

File: docs/user-research/research-plan-phase1.md [hunk 13]
```diff
@@ -1,79 +1,226 @@
...
+1. Can you articulate the problem that you think this project is trying to solve?
+
+### Collaboration
+
+1. Does your team collaborate with any other team currently?
+    * If so, who do you collaborate with and how?
+    * If so, what are some of the biggest challenges to effective collaboration?
+1. How do you collaborate with SREs/DevOps/Infrastructure practitioners?
+
+### Architecture and Data
```
[the remark this answers is not in the document]
Comment (kilbergr) [reply docs/user-research/research-plan-phase1.md:81]: Oooh definitely. Smart to ask that last one before "Do you know when you have inconsistent data" especially.

File: docs/user-research/research-plan-phase1.md [hunk 14]
```diff
@@ -1,79 +1,229 @@
...
 
-The ECLKC hosting infrastructure is hosted in AWS and serves the following applications:
+The ECLKC hosting infrastructure is hosted in AWS and serves the following applications/teams:
 
 * ECLKC
+* HSICC
 * National Centers
-  * iDP
-  * (more)
+  * iDP Learning Management System (includes iLookOut)
```
Comment (eeeady) [inline docs/user-research/research-plan-phase1.md:19]:
```suggestion
  * iPD Learning Management System (includes iLookOut)
```

**Review by eeeady — APPROVED** [review]
Just one typo fix. Thanks Rebecca!

**Pull request merged**